articles/quickstart/webapp/aspnet-core/_includes/_login.md

```diff
@@ -270,7 +256,7 @@ When the user selects the **Login** button, the OIDC middleware redirects them t
 
 ## Obtain an Access Token for Calling an API
 
-If you want to call an API from your MVC application, you need to obtain an Access Token issued for the API you want to call. To obtain the token, pass an additional `audience` parameter containing the API identifier to the Auth0 authorization endpoint.
+If you want to call an API from your MVC application, you need to obtain an Access Token issued for the API you want to call. To obtain the token, pass an additional `audience` parameter containing the API identifier to the Auth0 authorization endpoint, which you can get from the API's [Application Settings](${manage_url}/#/applications).
```

Comment:
I think this URL is wrong.

```suggestion
If you want to call an API from your MVC application, you need to obtain an Access Token issued for the API you want to call. To obtain the token, pass an additional `audience` parameter containing the API identifier to the Auth0 authorization endpoint, which you can get from the API's [Settings](${manage_url}/#/apis)
```

Reply:
I've fixed this locally rather than accepting the suggestion, as I wanted to tweak a couple of other things.

Reply:
Yes sounds good!